File: evo.funders.commonmessages/v1/DotNet/Models/FinancialStatus.cs
using Newtonsoft.Json;

namespace AzureFunderCommonMessages.DotNet.Models
{
    public class FinancialStatus
    {
        [JsonProperty("affordability", Required = Required.DisallowNull, NullValueHandling = NullValueHandling.Ignore)]
        public string? Affordability { get; set; } = "";

        [JsonProperty("affordabilityMonthlyMortgageRentContribution", Required = Required.Default,
            NullValueHandling = NullValueHandling.Include)]
        public long? AffordabilityMonthlyMortgageRentContribution { get; set; } = 0;

        [JsonProperty("affordabilityMonthlyOtherExpenditure", Required = Required.DisallowNull,
            NullValueHandling = NullValueHandling.Ignore)]
        public long? AffordabilityMonthlyOtherExpenditure { get; set; } = 0;

        [JsonProperty("affordabilityReason", Required = Required.DisallowNull,
            NullValueHandling = NullValueHandling.Ignore)]
        public string? AffordabilityReason { get; set; } = "";

        [JsonProperty("annualGrossIncome", Required = Required.DisallowNull,
            NullValueHandling = NullValueHandling.Ignore)]
        public long? AnnualGrossIncome { get; set; } = 0;

        [JsonProperty("netMonthlyIncome", Required = Required.DisallowNull,
            NullValueHandling = NullValueHandling.Ignore)]
        public double? NetMonthlyIncome { get; set; } = 0;
    }
}

File: evo.funders.commonmessages/v1/PHP/src/Models/FinancialStatus.php
<?php

namespace AzureFunderCommonMessages\PHP\Models;

use AzureFunderCommonMessages\PHP\Traits\JsonSerializer;

class FinancialStatus implements \JsonSerializable
{
	use JsonSerializer;
	protected ?int $annualGrossIncome;
	protected ?string $affordability;
	protected ?string $affordabilityReason;
	protected ?int $affordabilityMonthlyMortgageRentContribution;
	protected ?int $affordabilityMonthlyOtherExpenditure;
	protected ?float $netMonthlyIncome;

	public function getAnnualGrossIncome(): ?int
	{
		return $this->annualGrossIncome;
	}

	public function getAffordability(): ?string
	{
		return $this->affordability;
	}

	public function getAffordabilityReason(): ?string
	{
		return $this->affordabilityReason;
	}

	public function getAffordabilityMonthlyMortgageRentContribution(): ?int
	{
		return $this->affordabilityMonthlyMortgageRentContribution;
	}

	public function getAffordabilityMonthlyOtherExpenditure(): ?int
	{
		return $this->affordabilityMonthlyOtherExpenditure;
	}

	public function getNetMonthlyIncome(): ?float
	{
		return $this->netMonthlyIncome;
	}

	public function setAnnualGrossIncome(?int $annualGrossIncome): self
	{
		$this->annualGrossIncome = $annualGrossIncome;
		return $this;
	}

	public function setAffordability(?string $affordability): self
	{
		$this->affordability = $affordability;
		return $this;
	}

	public function setAffordabilityReason(?string $affordabilityReason): self
	{
		$this->affordabilityReason = $affordabilityReason;
		return $this;
	}

	public function setAffordabilityMonthlyMortgageRentContribution(?int $affordabilityMonthlyMortgageRentContribution): self
	{
		$this->affordabilityMonthlyMortgageRentContribution = $affordabilityMonthlyMortgageRentContribution;
		return $this;
	}

	public function setAffordabilityMonthlyOtherExpenditure(?int $affordabilityMonthlyOtherExpenditure): self
	{
		$this->affordabilityMonthlyOtherExpenditure = $affordabilityMonthlyOtherExpenditure;
		return $this;
	}

	public function setNetMonthlyIncome(?float $netMonthlyIncome): self
	{
		$this->netMonthlyIncome = $netMonthlyIncome;
		return $this;
	}
}
